OHM-Trade-Agent-v1/app/services/chief_alert_notifier.py

```
@@ -396,13 +409,58 @@ def send_trade_plan(
+                    trade_id=trade_id,
+                    message=message,
+                    candidate=candidate,
+                    plan=plan,
+                    action=action,
+                    direction=direction,
+                    identity=identity,
+                    fingerprint=key,
+                    reason=f"TRACKING_PENDING:{type(exc).__name__}",
+                )
+            except Exception:
+                pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)